### ISSUE TITLE
Add push on :+1: from submitter

### ISSUE THREAD

**Issue by larister** [marmelab/sedy#71]
### Description

Shamelessly pinched from @unholiness here: https://news.ycombinator.com/item?id=13753987

> Rather than triggering Sedy immediately on a reviewer comment, I'd like the trigger to be the original requester reacting to the comment with a thumbs-up. The requester knows what they're trying to say, and they should decide if the changes get made.

**Comment by Kmaschta** [comment]
I read that but I'm not convinced.

The point of Sedy is to prevent the original requester to switch context and take care of the review.
Moreover, the reviewer already can commit without the original requester permission (except it is a PR from fork and the requester had uncheck the permission to write on the branch).

**Comment by jpetitcolas** [comment]
Same as @Kmaschta. 

> The requester knows what they're trying to say, and they should decide if the changes get made.

The change requester also knows what he is doing, and hence, we can trust him IMO. :)

**Comment by larister** [comment]
Fair enough, I think it can change from company to company; perhaps it could be a setting if/when there's a .yml file? Regardless, thanks for your fine work and thanks for sharing 🙌

**Comment by Kmaschta** [comment]
Enable Sedy configuration with a `.sedy.yml` file can be a very nice feature, I agree!
